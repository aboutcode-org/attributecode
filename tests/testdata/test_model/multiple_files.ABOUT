about_resource: .
name: multiple_files

notice_file: NOTICE, NOTICE1
author_file: AUTHOR
changelog_file:

licenses:
    - key: lgpl-2.1
	  file: COPYING, COPYING.LESSER