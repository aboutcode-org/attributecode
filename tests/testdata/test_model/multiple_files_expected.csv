about_resource,name,notice_file,author_file,changelog_file,license_key,license_file
/test_model/,multiple_files,"NOTICE, NOTICE1",AUTHOR,,lgpl-2.1,"COPYING, COPYING.LESSER"
